lib/resources/dstu2/condition.yaml

```diff
@@ -139,12 +139,12 @@ fields:
     }
   binding:
     description: A category assigned to the condition.
-    note: Currently `diagnosis` and `problem` category codes are supported. Code `diagnosis` is bound to condition category codes and `problem` is bound to argonaut extenstion codes terminology system shown below.
+    note: Category codes `diagnosis`, `problem` and `health-concern` are currently supported for search and retrieve functionality. Code `health-concern` is currently not supported for conditions add and update functionality.
```

Comment:
This note is unclear. Can drop the "currently"s and it's unclear what category codes are supported for create and update. Diagnosis and problem are explicitly included out for search and retrieve. Is the exclusion of health-concern from create and update implying that diagnosis and problem are supported?